sound/soc/sof/intel/hda-dai.c

```diff
@@ -422,56 +422,176 @@ static struct snd_soc_cdai_ops sof_probe_compr_ops = {
 struct snd_soc_dai_driver skl_dai[] = {
 {
 	.name = "SSP0 Pin",
+	.playback = {
+		.stream_name = "ssp0 Tx",
+		.channels_min = 2,
+		.channels_max = 2,
+	},
+	.capture = {
+		.stream_name = "ssp0 Rx",
+		.channels_min = 2,
+		.channels_max = 2,
+	},
```

Comment:
Should we have 2 macros for these - one for playback and one for capture?

Reply:
> macros

@lyakh Do you mean create new macros for SSP?

Reply:
Yeah, I think there's quite a bit of repetition in the declarations now, so would be more readable with a few helper macros (just like the SND_SOC_DAILINK_REG in machine drivers).

Reply:
yes, that would be a good thing.